apps/hash-api/src/graph/system-types.ts

```diff
@@ -10,6 +10,11 @@ import { logger } from "../logger";
 import { ImpureGraphContext } from "./index";
 import { entityTypeInitializer, propertyTypeInitializer } from "./util";
 
+/**
+ * IF YOU EDIT THIS FILE in a way which affects the number or structure of system types,
+ * run `yarn generate-system-types` to update their TypeScript representation
```

Comment:
Ideally we would have a CI check that this has been run (similar to the CI check where we check the diff between `yarn.lock` files after `yarn install`). Maybe we can add a `@todo` for this?

Reply:
For the Graph API's OpenAPI specs we do a similar thing. It's part of our testing suite and after all tests have run we enforce an empty `git diff`.

Reply:
Todos added in https://github.com/hashintel/hash/pull/2852/commits/1ac1e2d4eebf4ea9f88090177ee4f3ff39a140a3, H-308